### DISCUSSION TITLE
extractCSS not working in Nuxt 3

### DISCUSSION THREAD

**Discussion by proveanything** [nuxt/nuxt#23273] [Questions]
Sorry to ask this - I've searched high and low and I'd be interested to know if anyone has managed this. I can find a lot of comments on Nuxt 2....but in Nuxt 3 I'm really struggling to extract my CSS from being  directly embedding into the HTML - and to move it to an external file that will benefit from caching.

The docs give a webpack example: https://nuxt.com/docs/api/configuration/nuxt-config#extractcss  and say it's true by default
I'm using vite/nitro and I've tried all sort of other options and suggestions and so far nothing has served to get the css to be extracted out to a separate file. It would be good to just know if anyway actually has this working at all!

I've seen people say that the "default" is to extract it  (i.e. https://github.com/nuxt/framework/pull/4388) - in which case I've no idea why mine is inline - but then I've also seen comments in the Nuxt docs that say all css will be "inline rendered in the HTML"  (https://nuxt.com/docs/getting-started/styling#the-css-property)    

Thanks

Glenn

**Comment by Hebilicious** [comment]
@proveanything Have you tried https://nuxt.com/docs/api/configuration/nuxt-config#inlinessrstyles
